codecov_auth/admin.py

```diff
@@ -310,6 +311,66 @@ class AccountAdmin(AdminMixin, admin.ModelAdmin):
         "is_delinquent",
     ]
 
+    @admin.action(description="Link Users to Account")
+    def link_users_to_account(self, request, queryset):
+        for account in queryset:
+            all_plan_activated_user_ids = set()
+            for org in account.organizations.all():
+                all_plan_activated_user_ids.update(set(org.plan_activated_users))
+
+            all_plan_activated_users = Owner.objects.filter(
+                ownerid__in=all_plan_activated_user_ids
+            ).prefetch_related("user")
+            non_student_count = all_plan_activated_users.exclude(student=True).count()
+            total_seats_for_account = account.plan_seat_count + account.free_seat_count
```

Comment:
What's a free seat count? I see it's defaulted to 0, is this for like free plans?

Reply:
it's a thing that exists on Owners currently (named `free`), but it's a way for us to give Orgs additional seats not in the count they pay for (`plan_seat_count`) 🤷

Reply:
damn, was not aware of this